app/controllers/spree/admin/payments_controller.rb

```diff
@@ -35,15 +35,16 @@ def create
             return
           end
 
-          authorize_stripe_sca_payment
-
           if @order.completed?
-            @payment.process!
+            authorize_stripe_sca_payment
+            @payment.process_offline!
             flash[:success] = flash_message_for(@payment, :successfully_created)
 
             redirect_to spree.admin_order_payments_path(@order)
           else
             OrderWorkflow.new(@order).complete!
+            authorize_stripe_sca_payment
```

Comment:
This is also quite smelly to be in the controller. I see it has nothing to do with it and it'd be better off in a separate service. Again, stuff for upcoming iterations.

Reply:
Agreed. I'll mark this as resolved since I think this will be addressed in https://github.com/openfoodfoundation/openfoodnetwork/pull/7809